### PR TITLE
Fix horizontal movement of the snake

### LINKED ISSUE #156: Fix the snake horizontal vs vertical inconsistent speed
## Description
The movement of the snake clearly seems to be inconsistent. The snake speed horizontally is significantly slow in comparison with the snake vertical movement. 
This issue can clearly disturb the player experience of playing the game.

## How it should be
The speed of the snake should be the same (or close to), horizontally and vertically.

## Alternatives 
Seems that the snake moves one character by time, horizontally and vertically. 
Since the distance between two characters side-by-side in the same line is significantly smaller than two characters side-by-side in the same column, the snake could move one blank line more when moving horizontally.

### PR DESCRIPTION
**Pull request by yuji-teoi**
Closes #156 .

This PR proposes the following changes:
- Fix horizontal movement of the snake to properly move at the same distance horizontally and vertically.
- Change some variables related to these movements, including the position of the snake and the power block.

### CODE REVIEW

File: src/ttsnake.c [hunk 1]
```diff
@@ -326,8 +326,12 @@ void init_game () {
 /* Generates energy_block[0] coordinates randomly. */
 
 void generate_energy_block () {
-  energy_block[0].x = (rand() % (NCOLS - 2)) + 1;
-  energy_block[0].y = (rand() % (NROWS - 2)) + 1;
+  energy_block[0].x = (rand() % (NCOLS - 2)) + HORIZONTAL_MOVE;
+  energy_block[0].y = (rand() % (NROWS - 2)) + VERTICAL_MOVE;
+
+  /* Verifies if the energy is in a pair position, 'cause the snake moves 2 pos horizontally */
+  if ((energy_block[0].x)%2 != 0)
```
Comment (gabriel-libardi) [inline src/ttsnake.c]: Please utilize the coding style that was agreed upon (see discussion at #13). 
```suggestion
  if ((energy_block[0].x)%2 != 0) {
    energy_block[0].x -= 1;
  }
}
```

File: src/ttsnake.c [hunk 2]
```diff
@@ -194,7 +194,7 @@ int read_scenes (char *dir, char *data_dir, scene_t** scene, int nscenes) {
 
 
 /* Draw a the given scene on the screen. Currently, this iterates through the
-   scene matrix outputig each caracter by means of indivudal puchar calls. One
+   scene matrix outputig each caracter by means of individudal puchar calls. One
```
Comment (gabriel-libardi) [inline src/ttsnake.c:197]: Replaced a typo with another typo.

**Review by gabriel-libardi — CHANGES_REQUESTED** [review]
There are just a couple of minor issues with your code. Other than that, it is working fairly well (and the code is more readable).

File: src/ttsnake.c (hunk 1 above)
Comment (yuji-teoi) [reply src/ttsnake.c]: commited

**Pull request closed without merging**